package.json

```diff
@@ -23,7 +23,7 @@
     "prettier": "^3.5",
     "solady": "0.0.208",
     "solarray": "github:evmcheb/solarray#a547630",
-    "solhint": "^5.1"
+    "solhint": "^6.0.0"
```

Comment:
I can see that you bumped solhint. I would like you to revert it and bump it separately. The new version introduced several new changes that should be implemented when bumping it up.

See https://github.com/sablier-labs/command-center/issues/165